### PR TITLE
Fix local build issues

### PR DESCRIPTION
**Pull request by mikedotexe**
The commit names say it all, but basically making sure the README has the proper instructions for all users, new and seasoned.
The script was trying to deploy the compiled wasm from the wrong directory.
Light changes on the HTML to make it easier to copy/paste to test.
Also, since we're using `await` added that into the HTML. Same basic idea, but just the more proper way to do it that's in line with how we do it elsewhere.

### CODE REVIEW

**Comment by willemneal** [comment]
I tried it out with gitpod and it works.  I thought I had the settings correct for gitpod to add a button, but if you want to try it: https://gitpod.io/#https://github.com/nearprotocol/token-contract-as/pull/1

One other note, is that we can make gitpod globally install near-shell, and you can run with `yarn near ...`

**Comment by mikedotexe** [comment]
> I tried it out with gitpod and it works. I thought I had the settings correct for gitpod to add a button, but if you want to try it: https://gitpod.io/#https://github.com/nearprotocol/token-contract-as/pull/1
> 
> One other note, is that we can make gitpod globally install near-shell, and you can run with `yarn near ...`

Nice thought. After a couple iterations, I went with aliasing it. Then it will really feel the same when the user develops locally and in GitPod. in both it's just `near login` instead of `yearn near login` in one. Thanks, that led me down a good path.

File: package.json [hunk 1]
```diff
@@ -4,23 +4,23 @@
...
     "deploy:pages": "gh-pages -d src",
     "deploy": "yarn build && yarn deploy:contract && yarn deploy:pages",
     "prestart": "yarn build && yarn deploy:contract",
-    "start": "serve src",
-    "asp:ci": "asp",
+    "start": "parcel src/index.html",
     "asp": "asp --verbose",
-    "test": "yarn asp:ci && yarn jest",
-    "jest": "yarn build && jest test --env=near-shell/test_environment"
+    "test": "yarn asp"
```
Comment (willemneal) [inline package.json]: Why did you remove jest?  Shouldn't we test both environments?

Comment (mikedotexe) [reply package.json]: glad you saw that, I was able to run asp locally as well. when you say "both environments" do you mean Gitpod and local?
